types/src/v0/traits.rs

```diff
@@ -467,6 +467,7 @@ pub trait SequencerPersistence: Sized + Send + Sync + 'static {
             state,
             validated_state,
             view,
+            highest_voted_view,
```

Comment:
Can we change v0 struct like this? @sveitser 

We need this to placate the tests, but not sure if we should edit the v0

Reply:
I think its fine to change it since we don't define it at sequencer side and serialize it.